app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -277,7 +357,51 @@ class Overview extends React.Component {
             clusters={clusters}
             transformationMappings={transformationMappings}
             createInfraMappingClick={showMappingWizardAction}
+            inProgressRequestsTransformationMappings={inProgressRequestsTransformationMappings()}
+            showDeleteConfirmationModalAction={
+              showDeleteConfirmationModalAction
+            }
+            setMappingToDeleteAction={setMappingToDeleteAction}
           />
+          <Modal
```

Comment:
would you mind moving this Modal to another stateless functional component? i.e. `components/DeleteMappingConfirmationModal` ? I think extracting this Modal JSX and relevant logic there in another component would be nice...

Will try to test this out on my side today...